scala/private/rule_impls.bzl

```diff
@@ -1202,13 +1209,20 @@ def scala_test_impl(ctx):
         ])
         coverage_runfiles = ctx.files._jacocorunner + ctx.files._lcov_merger + coverage_replacements.values()
 
+    # jvm_flags passed in on the target override scala_test_jvm_flags passed in on the
+    # toolchain
+    final_jvm_flags = _first_non_empty(
+        ctx.attr.jvm_flags,
```

Comment:
why don't we need `_expand_location` here?

Reply:
It wasn't wrapped before, so I left it unwrapped, but thinking about this more, we probably want `final_jvm_flags` itself wrapped.